src/Compute/Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -5312,4 +5312,54 @@ function Test-VirtualMachineScaleSetDefaultImgWhenStandard
         # Cleanup
         Clean-ResourceGroup $rgname;
     }
-}
\ No newline at end of file
+}
+
+<#
+.SYNOPSIS
+    Create a VMSS 
+    Test ResilientVMCreationPolicy
+    Test ResilientVMDeletionPolicy
+#>
+function Test-ResiliencyPolicyVMSS
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName
+
+    try
+    {
+        # Common
+        $loc = Get-ComputeVMLocation;
+        New-AzResourceGroup -Name $rgname -Location $loc -Force;
+
+        # New VMSS Parameters
+        $vmssName = 'vmssResiliencyPolicy' + $rgname;
+
+        $adminUsername = 'Foo12';
```

Comment:
you could use Get-ComputeTestResourceName here instead. Might be better security wise.

Reply:
how is it better security wise? The resource will be deleted after the test is ran.

Reply:
Added